android/app/src/main/java/ru/spbstu/amd/learnbraille/screens/practice/PracticeFragment.kt

```diff
@@ -52,6 +62,16 @@ class PracticeFragment : Fragment() {
             )
         }
 
+        // init serial connection with Braille Trainer
+        @SuppressLint("WrongConstant") // permit `application.getSystemService(USB_SERVICE)`
+        val usbManager = application.getSystemService(USB_SERVICE) as UsbManager

```

Comment:
It may be a better way to move unsafe cast to Application extension function.

Reply:
I don't know what is an `application extension function`, but still it will be a patch anyway, so maybe let it be a patch as is?